test/API/H5_api_attribute_test.c

```diff
@@ -1130,9 +1130,6 @@ test_create_attribute_invalid_params(void)
             }
 
             PASSED();
-#else
-            SKIPPED();
-            PART_EMPTY(H5Acreate_invalid_aapl);
```

Comment:
You probably want to leave the SKIPPED section in this test until it's fixed or it will mess up the output formatting for the tests.

Reply:
Okay, I can leave it.